legacy/aws-amplify-react-native/README.md

```diff
@@ -3,3 +3,952 @@
+
+When you are using `email` as the username:
+
+```js
+import { withAuthenticator, Authenticator } from 'aws-amplify-react';
+
+// When using Authenticator
+class App {
+  // ...
+
+  render() {
+    return (
+      <Authenticator usernameAttributes='email'/>
+    );
+  }
+}
+
+export default App;
+
+// When using withAuthenticator
+class App2 {
+  // ...
+}
+
+export default withAuthenticator(App2, { usernameAttributes: 'email' });
+```
+
+When you are using `phone number` as the username:
+
+```js
+import { Authenticator, withAuthenticator } from 'aws-amplify-react';
+
+class App {
+  // ...
+
+  render() {
+    return (
+      <Authenticator usernameAttributes='phone_number'/>
+    );
+  }
+}
+
+export default App;
+
+// When using withAuthenticator
+class App2 {
+  // ...
+}
+
+export default withAuthenticator(App2, { usernameAttributes: 'phone_number' });
+```
+
+**Note:** If you are using custom signUpFields to customize the `username` field, then you need to make sure either the label of that field is the same value you set in `usernameAttributes` or the key of the field is `username`.
+
+For example:
+
+```js
+import React, { Component } from 'react';
+import Amplify from 'aws-amplify';
+import awsconfig from './aws-exports';
+import { withAuthenticator } from 'aws-amplify-react';
+
+Amplify.configure(awsconfig);
+
+class App extends Component {}
+
+const signUpConfig = {
+  header: 'My Customized Sign Up',
+  hideAllDefaults: true,
+  defaultCountryCode: '1',
+  signUpFields: [
+    {
+      label: 'My user name',
+      key: 'username',
+      required: true,
+      displayOrder: 1,
+      type: 'string',
+    },
+    {
+      label: 'Password',
+      key: 'password',
+      required: true,
+      displayOrder: 2,
+      type: 'password',
+    },
+    {
+      label: 'PhoneNumber',
+      key: 'phone_number',
+      required: true,
+      displayOrder: 3,
+      type: 'string',
+    },
+    {
+      label: 'Email',
+      key: 'email',
+      required: true,
+      displayOrder: 4,
+      type: 'string',
+    },
+  ],
+};
+const usernameAttributes = 'My user name';
+
+export default withAuthenticator(App, {
+  signUpConfig,
+  usernameAttributes,
+});
+```
+
+---
+
+# API - Connect
+
+The API category provides React Native components for working with GraphQL data using the Amplify GraphQL client.
+
+The `<Connect/>` component is used to execute a GraphQL query or mutation. You can execute GraphQL queries by passing your queries in `query` or `mutation` attributes
+
+## Subscription
+
+For subscriptions, you can use the `subscription` and `onSubscriptionMsg` attributes to enable subscriptions
+
+## Mutations
+
+For mutations, a `mutation` function needs to be provided with the `Connect` component. A `mutation` returns a promise that resolves with the result of the GraphQL mutation.
+
+---
+
+# Storage
+
+## S3 Album
+
+`S3Album` renders a list of `S3Image` and `S3Text` objects:
+
+![Image](/images/S3Album_and_code.png)
```

Comment:
Does this exist?

Reply:
@reesscot had to do the same for #790

Reply:
Yes, will update now

Reply:
Fixed:
![Screen Shot 2021-11-18 at 5 19 28 PM](https://user-images.githubusercontent.com/6165315/142517464-0ea7dc5a-ffe8-4f1b-9ea4-f2d18587d05d.png)